boards: frdm_mcxn947: turn OV7670 into a shield (#89131)
zephyrproject-rtos/zephyr: boards/shields/dvp_20pin_ov7670/doc/index.rst

(Digital Video Port), aka parallel interface.

Only 18 pins out of the 20-pin connector are present

It was originally produced by `Arducam`_ but is discontinuited, and now `Olimex`_ provides it.

Pins assignment
===============

+-----+--------------+-----+--------------+
| Pin | Function     | Pin | Function     |
+=====+==============+=====+==============+
| 1   | 3V3          | 2   | GND          |
+-----+--------------+-----+--------------+
| 3   | SCL          | 4   | SDA          |
+-----+--------------+-----+--------------+
| 5   | VS           | 6   | HS           |
+-----+--------------+-----+--------------+
| 7   | PCLK         | 8   | XCLK         |
+-----+--------------+-----+--------------+
| 9   | D7           | 10  | D6           |
+-----+--------------+-----+--------------+
| 11  | D5           | 12  | D4           |
+-----+--------------+-----+--------------+
| 13  | D3           | 14  | D2           |
+-----+--------------+-----+--------------+
| 15  | D1           | 16  | D0           |
+-----+--------------+-----+--------------+
| 17  | POWER_EN     | 18  | POWER_DOWN   |
+-----+--------------+-----+--------------+

Requirements
************

> This shield can be used with any board which provides a 18 or 20-pin header spread over two rows

Reviewer: ```suggestion
This shield can be used with any board that provides an 18 or 20-pin header spread over two rows
```
